tools/sigh.ts

```diff
@@ -63,6 +63,7 @@ import * as AstNode from '../../runtime/manifest-ast-nodes.js';
 };
 
 const steps: {[index: string]: ((args?: string[]) => boolean)[]} = {
+  lsp: [peg, build, webpackTools, lsp],
```

Comment:
`lsp` is pretty opaque; I have no idea what to expect when running this :)

Reply:
Renamed to languageServer (and added brief comment in function to explain further)